### PR TITLE
[3.0] nova: Allow defining on which network live migration happens (bsc#990664)

### PR DESCRIPTION
**Pull request by bhdn**
This is a backport of pull request #466 .

### CODE REVIEW

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by MaximilianMeister** [comment]
@bhdn I retriggered the gating, It was stuck somehow...

+1

**Comment by aplanas** [comment]
Can you rebase? Sorry.

**Comment by MaximilianMeister** [comment]
@bhdn needs rebase again

**Comment by bhdn** [comment]
Updated.

**Comment by nicolasbock** [comment]
Gating seems to be stuck.

**Comment by rsalevsky** [comment]
Can you rebase?

**Comment by bhdn** [comment]
Pushed.

**Comment by vuntz** [comment]
Triggered Jenkins...

**Comment by vuntz** [comment]
Worth mentioning: the commit id mentioned for the cherry-pick is not correct anymore (see https://github.com/crowbar/crowbar-openstack/pull/466/commits)

**Comment by toabctl** [comment]
needs rebase

**Comment by bhdn** [comment]
Pushed

File: chef/data_bags/crowbar/migrate/nova/046_add_migration_network.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def upgrade(ta, td, a, d)
```
Comment (vuntz) [inline chef/data_bags/crowbar/migrate/nova/046_add_migration_network.rb]: Looks like revision 46 was already taken :/ This needs a revision bump, so I guess this needs a different revision.

Comment (bhdn) [inline chef/data_bags/crowbar/migrate/nova/046_add_migration_network.rb]: Ouch, thanks for pointing.

**Comment by rsalevsky** [comment]
https://github.com/crowbar/crowbar-openstack/pull/479/commits/e1f0542881223b911919d2ba383c13b12bfd66ca#diff-64e5bfb5f50eb1efa946a22652b005d6R103 needs to be 47

**Comment by vuntz** [comment]
Triggered gating...

**Pull request merged**